app/renderer/components/navigation/navigator.js

```diff
@@ -150,6 +152,172 @@ class Navigator extends React.Component {
     ipc.off(messages.SHORTCUT_ACTIVE_FRAME_FORWARD, this.onForward)
   }
 
+  // BEM Level: navigator__menuBarAndNavigationBar__navigationBarWrapper
+  get navigationBarWrapper () {
```

Comment:
can we move this function back into renderer. I agree that some parts is good to move into getters, but we should still keep some code in the renderer. This was is easier to see what we are doing.